flink/v2.0/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergStreamWriter.java

```diff
@@ -46,13 +45,12 @@ class IcebergStreamWriter<T> extends AbstractStreamOperator<FlinkWriteResult>
   IcebergStreamWriter(String fullTableName, TaskWriterFactory<T> taskWriterFactory) {
     this.fullTableName = fullTableName;
     this.taskWriterFactory = taskWriterFactory;
-    setChainingStrategy(ChainingStrategy.ALWAYS);
```

Comment:
why remove this line?

Reply:
This isn't supported anymore. Forced chaining can only be controlled by the Flink runtime. The operator will still be chained though, unless chaining is globally disabled.